pyvrp/search/neighbourhood.py

```diff
@@ -24,7 +24,7 @@ class NeighbourhoodParams:
         Penalty weight given to the minimum time warp aspect of the proximity
         calculation. A large time warp indicates the clients are far apart in
         duration/time.
-    nb_granular
+    num_neighbours
```

Comment:
Could've updated this to `num_granular`, but I think `num_neighbours` is much more descriptive.